Source/src/components/ComponentMesh.cpp

```diff
@@ -100,16 +129,18 @@ void Hachiko::ComponentMesh::UpdateSkinPalette(float4x4* palette) const {
         for (unsigned i = 0; i < mesh->num_bones; ++i)
         {
             const ResourceMesh::Bone& bone = mesh->bones[i];
-            const GameObject* bone_node = node_cache[i];
+            /* const GameObject* bone_node = node_cache[i];
```

Comment:
This is another improve that apears on the diapo called Cache bones. @baransrc if you want i can dedicate time to implemnt i think it's easy and improve the performance.